coordinator/src/routes/prescription/post-prepare-message.ts

```diff
@@ -18,6 +18,8 @@ export default [
         const isSmokeTest = request.headers["x-smoke-test"]
         const contentType = isSmokeTest ? CONTENT_TYPE_JSON : CONTENT_TYPE_FHIR
         const response = translator.convertFhirMessageToSignedInfoMessage(requestPayload)
+        request.log("audit", {messageType: "Inbound FHIR message", payload: requestPayload})
+        request.log("audit", {messageType: "Outbound signed info message", outbound: response})
```

Comment:
am i right in thinking this will make a call to hapi's own logger and not the pino instance / does that matter?

Reply:
It goes through Hapi, but pino acts as the default logger anyway